### ISSUE TITLE
Enhance the training data and improve the accuracy

### ISSUE THREAD

**Issue by ch3ssd** [ch3ssd/ChessAI#4]
The current model's predictive accuracy is limited due to insufficient data. A robust and automated visual data generation process is needed to enrich the training dataset.

**Comment by draw-four** [comment]
Check out data augmentation. Though your resultant data set might be prohibitively large to store on Github directly (checkout Git Large File Storage) you can use a number of image processing techniques such as rotating cropping or shearing, adding noise or aberration and any combination of the above to generate a large quantity of useful data from existing images.

**Comment by draw-four** [comment]
Additionally, ensure that insufficient data is your reason. If possible with FastAI, lower the amount of layers/neurons you have, this may be a case of overfitting. The most common symptom of overfitting is that training accuracy is extremely high, but testing accuracy is markedly lower.
